ui/lib/pki/addon/components/page/pki-tidy-status.ts

```diff
@@ -137,10 +137,10 @@ export default class PkiTidyStatusComponent extends Component<Args> {
   @task
   @waitFor
   *cancelTidy() {
-    // TODO: make a custom adapter method when Claire merges her form work!
     try {
-      const adapter = this.store.adapterFor('application');
-      yield adapter.ajax(`/v1/${this.secretMountPath.currentPath}/tidy-cancel`, 'POST');
+      const tidyAdapter = this.store.adapterFor('pki/tidy');
+      yield tidyAdapter.cancelTidy(this.secretMountPath.currentPath);
+      this.confirmCancelTidy = false;
```

Comment:
Nice work! I'd either move this line (143) to a `finally` block or before the `try...catch` all together so that the confirmation modal closes whether or not the request is successful 

```js
try {
 const tidyAdapter = this.store.adapterFor('pki/tidy');
 yield tidyAdapter.cancelTidy(this.secretMountPath.currentPath);
} catch (error) {
 this.flashMessages.danger(errorMessage(error));
} finally {
 this.confirmCancelTidy = false;
}